src/service/viewer-impl.js

```diff
@@ -499,6 +511,34 @@ export class Viewer {
     return this.isWebviewEmbedded_;
   }
 
+  /**
+   * Whether the document is using Chrome Custom Tabs.
+   * @return {boolean}
+   */
+  isCustomTab() {
+    return this.isCustomTab_;
+  }
```

Comment:
Remove unnecessary getter.

Reply:
Done

Reply:
Added back to support another PR being worked on by @RJSumi (related to Client ID behavior when CCT is used).